pkg/convert/pot.go

```diff
@@ -76,12 +76,14 @@ func FromPot(b []byte, original bool) (model.Translation, error) {
 		getStatus = func(_ pot.MessageNode) model.MessageStatus { return model.MessageStatusTranslated }
 	}
 
-	convert := func(v pot.MessageNode) string { return convertToMessageFormatSingular(singularValue(v)) }
+	convert := func(v pot.MessageNode) string {
+		return messageformat.ToMessageFormat2(singularValue(v))
+	}
 
 	if po.Header.PluralForms.NPlurals == pluralCountLimit {
 		convert = func(v pot.MessageNode) string {
 			if v.MsgIDPlural == "" {
-				return convertToMessageFormatSingular(singularValue(v))
+				return messageformat.ToMessageFormat2(singularValue(v))
```

Comment:
Interesting function name. Are you converting message format to message format 2?

Reply:
source(extracted) string to message format

Reply:
To make the intent of the function more obvious, perhaps - mf2.MessageFromString()?

Reply:
There are common interfaces we can use to convey serialisation - MarshalText & UnmarshalText?